experimental/dds/tree2/src/test/feature-libraries/sequence-field/sequenceFieldToDelta.spec.ts

```diff
@@ -237,10 +286,14 @@ describe("SequenceField - toDelta", () => {
 	});
 
 	it("modify and delete => delete", () => {
-		const changeset = [Mark.delete(1, brand(0), { changes: childChange1 })];
-		const mark: Delta.Delete = {
-			type: Delta.MarkType.Delete,
+		const changeset: TestChangeset = [
+			{ type: "Delete", id: brand(0), count: 1, changes: childChange1 },
+		];
```

Comment:
Try to use the Mark.delete instead of making changeset literals by hand. This may have been from a bad merge.